src/api/apiFetch.js

```diff
@@ -44,13 +44,14 @@ export const apiCall = async (
       }, 5000);
     }
 
-    const response = await apiFetch(authObj, route, params, noTimeout);
+    const response = await apiFetch(auth, route, params, noTimeout);
     if (response.status === 401) {
       // TODO: httpUnauthorized()
       throw Error('Unauthorized');
     }
 
     if (!response.ok) {
+      console.error(response); // eslint-disable-line
```

Comment:
Because if no code handles it, often it is an unhandled promise rejection hard to track down.
It is removed for production anyways.